website/docs/reference/programmatic-invocations.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Programmatic invocations"
+---
+
+In v1.5, dbt-core added support for a first-ever Python API for programmatic invocations. The intent of this API is to provide **exact parity** with CLI functionality, callable from within a Python script or application.
+
+The entry point is a `dbtRunner` class that wraps around the `dbt-core` CLI and allows you to "invoke" CLI commands as Python methods. Each command returns a tuple of:
+- `results` (return type depends on command)
+- `success` (boolean): whether the command succeeded
+
+```python
+from dbt.cli.main import dbtRunner
+
+# initialize
+dbt = dbtRunner()
+
+# create CLI args as a list of strings
+cli_args = ['run', '--select', 'some_models']
+
+# run the command
+results: List[Any]
+success: boolean
```

Comment:
These need to be commented out for the script to run. Other than this, the example works out-of-the-box.
```suggestion
# results: List[Any]
# success: boolean
```

Reply:
I updated the example here now that `dbtRunner` returns `dbtRunnerResult`, instead of a tuple (changed in https://github.com/dbt-labs/dbt-core/pull/7242)